src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -431,18 +435,22 @@
 (defn navigation-bar
   []
   (let [logged-in?    (rf/sub [:multiaccount/logged-in?])
-        has-profiles? (boolean (rf/sub [:profile/profiles-overview]))]
+        has-profiles? (boolean (rf/sub [:profile/profiles-overview]))
+        root          (if has-profiles? :profiles :intro)]
     [quo/page-nav
      {:align-mid?   true
       :mid-section  {:type      :text-only
                      :main-text "Quo2 components preview"}
       :left-section {:icon     :i/close
                      :on-press (fn []
-                                 (when-not logged-in?
-                                   (theme/set-theme :dark))
-                                 (rf/dispatch [:navigate-back])
-                                 (when-not has-profiles?
-                                   (rf/dispatch [:open-modal :new-to-status])))}}]))
+                                 (cond
+                                   logged-in?
+                                   (rf/dispatch [:navigate-back])
+
+                                   :else
+                                   (do
+                                     (theme/set-theme :dark)
+                                     (rf/dispatch [:init-root root]))))}}]))
```

Comment:
Updated this function to fix a modal bug when the user navigates to the `New to status` screen from the `+` button in the "profiles" list screen.